lib/cib/cib_remote.c

```diff
@@ -195,18 +195,24 @@ cib_remote_perform_op(cib_t *cib, const char *op, const char *host,
     return rc;
 }
 
+struct remote_cb_data_s {
+    cib_t *cib;
+    int timeout_sec;
+};
```

Comment:
You don't need a new struct for this -- `cib_remote_opaque_t` is for remote-specific data